src/applications/disability-benefits/526EZ/actions/index.js

```diff
@@ -1,64 +1,158 @@
-/* eslint-disable no-unused-vars */
 import { apiRequest } from '../../../../platform/utilities/api';
-import { fetchInProgressForm } from '../../../common/schemaform/save-in-progress/actions';
 
-export const ITFStatuses = Object.freeze({
-  active: 'active',
-  claim_received: 'claim_received', // eslint-disable-line camelcase
-  duplicate: 'duplicate',
-  expired: 'expired',
+import existingITFData from '../tests/itfData';
+
+export const PRESTART_STATUS_SET = 'PRESTART_STATUS_SET';
+export const PRESTART_DATA_SET = 'PRESTART_DATA_SET';
+export const PRESTART_STATE_RESET = 'PRESTART_STATE_RESET';
+export const PRESTART_DISPLAY_RESET = 'PRESTART_DISPLAY_RESET';
+
+export const PRESTART_STATUSES = {
+  notAttempted: 'not-attempted',
   pending: 'pending',
-  incomplete: 'incomplete'
-});
-
-
-// TODO: replace mock once ITF endpoint setup
-export function submitIntentToFile(formConfig, onChange) {
-  const { intentToFileUrl, formId, migrations, prefillTransformer } = formConfig;
-  let ITFStatus = ITFStatuses.pending;
-  return dispatch => {
-
-    onChange(ITFStatus);
-
-    const delay = (ms) => new Promise((resolve, reject) => {
-      ITFStatus = ITFStatuses.active;
-      onChange({ ITFStatus });
-
-      // TODO: if the backend handles resubmission, this check can be removed
-      if (ITFStatus === ITFStatuses.active) {
-        setTimeout(resolve, ms);
-      } else {
-        const errorMessage = 'Network request failed';
-        onChange({
-          ITFStatus,
-          errorMessage
-        });
-        reject(errorMessage);
-      }
-    });
-
-    return delay(2000);
+  succeeded: 'succeeded',
+  failed: 'failed'
+};
+
+export const PRESTART_TYPES = {
+  retrieve: 'retrieve',
+  create: 'create'
```

Comment:
@cvalarida @markgreenburg I decided to split the create or retrieve information out of the status and store it along with the rest of the data that is used to generate the message for the user. Thanks for your thoughts on this.